### ISSUE TITLE
[BUG] - Minor: incorrect description in the Build the Neural Network tutorial

### ISSUE THREAD

**Issue by vaharoni** [pytorch/tutorials#3622]
### Add Link

https://docs.pytorch.org/tutorials/beginner/basics/buildmodel_tutorial.html

### Describe the bug

The tutorial has the following paragraph:
```
Calling the model on the input returns a 2-dimensional tensor with dim=0 corresponding to each output of 10 raw predicted values for each class, and dim=1 corresponding to the individual values of each output. 
```

I believe this is inaccurate. dim=0 should correspond to the batch dimension, and dim=1 should have a size of 10, corresponding to the logits of each class.

I'll be submitting a PR shortly with suggested copy, assuming you agree with my observation.

### Describe your environment

N/A

**Comment by Dhull442** [comment]
I agree, the wording was somewhat confusing too.
